contentctl/api.py

```diff
@@ -0,0 +1,124 @@
+from pathlib import Path
+from typing import Any, Union, Type
+#from contentctl import contentctl
+from contentctl.input.yml_reader import YmlReader
+from contentctl.objects.config import test_common, test_servers, test
```

Comment:
`test_common` is unused

Reply:
nit: again, not something for this MR, but just calling it out here, but, on the flip-side, class names should not be in snake-case for PEP8

Reply:
I removed test_common. You are correct about following Pep-8. I have addressed this for function names in the new file, but have not addressed it for imported classes.
For now, we will get better at new code following proper conventions.  We will eventually have an MR that cleans up existing code as well.